grim_test.go

```diff
@@ -212,6 +212,35 @@ func TestHookGetsLogged(t *testing.T) {
 
 }
 
+func TestShouldSkip(t *testing.T) {
```

Comment:
Excellent job with this test!

There is a case that should be added.  Its possible for `EventName` to be `issue` and `Action` to be `opened` (see [this](https://developer.github.com/v3/activity/events/types/#issuesevent) for details) and we shouldn't build in that case.